### ISSUE TITLE
Python: 1.3 Learning version control- GitHub terminologies

### ISSUE THREAD

**Issue by adityabisoi** [girlscript/winter-of-contributing#48]
### Description :
Explain the generally used GitHub terminologies (branch, upstream, pull, push, etc.)

### Note : 
- Changes should be made inside the `Python/Git_And_GitHub/` directory & `Python` branch.
-  Make necessary changes to `README.md` also.
-  Issue only for `GWOC'21` contributors.
-  Issue will be assigned on a **first come first serve basis**, **1 Issue  ==  1 PR.**  

#### Must Follow : [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of Conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) before start Contributing.

**Comment by kunal-kumar-chaudhary** [comment]
hey! i would like to work on this issue.

**Comment by harmankaler2000** [comment]
Hey! I would like to work on this issue. It would help me gain a better grasp on the topics whilst explaining it to others!

**Comment by adityabisoi** [comment]
@kunal-kumar-chaudhary Go ahead

**Comment by kunal-kumar-chaudhary** [comment]
> @kunal-kumar-chaudhary Go ahead

thank you, will remain in touch with you and will keep updating on the progress. will ask doubts later if i come across any problem.

**Comment by kunal-kumar-chaudhary** [comment]
> @kunal-kumar-chaudhary Go ahead

is there any sort of deadline?

**Comment by nayakaj** [comment]
@adityabisoi  we can share the link of google notes

**Comment by adityabisoi** [comment]
@kunal-kumar-chaudhary No, @nayakaj Read the contribution guidelines for details on contribution types.

**Comment by PVenkatArvind** [comment]
Greetings !!
I am interested, Can I contribute
Thanks & Regards,
P Venkat Arvind
Python Contributor

**Comment by lohith1392** [comment]
I would like to be assigned this issue
Peddi sai lohith
python batch-4

**Comment by adityabisoi** [comment]
@kunal-kumar-chaudhary Will you commit as documentation, audio or video?

**Comment by kunal-kumar-chaudhary** [comment]
> @kunal-kumar-chaudhary Will you commit as documentation, audio or video?

Documentation.

**Comment by adityabisoi** [comment]
This issue is open for AUDIO and VIDEO

**Comment by kunal-kumar-chaudhary** [comment]
> This issue is open for AUDIO and VIDEO

is it 'only' open for audio and video or 'also' open for audio and video?

**Comment by adityabisoi** [comment]
> > This issue is open for AUDIO and VIDEO
> 
> is it 'only' open for audio and video or 'also' open for audio and video?

Also.

**Comment by kunal-kumar-chaudhary** [comment]
> > > This issue is open for AUDIO and VIDEO
> > 
> > 
> > is it 'only' open for audio and video or 'also' open for audio and video?
> 
> Also.

I have created the pull request. kindly check it.

**Comment by adityabisoi** [comment]
@prathimacode-hub Is the mentor of this issue. Please reach out to her incase of any queries.

**Comment by kunal-kumar-chaudhary** [comment]
hey @prathimacode-hub ! i have created the pull request. please check it! #629

**Comment by prathimacode-hub** [comment]
@kunal-kumar-chaudhary, I am a mentor for this issue. If any queries, feel free to ask. If you had made a PR, Add up your PR links here.

**Comment by kunal-kumar-chaudhary** [comment]
> > @kunal-kumar-chaudhary, I am a mentor for this issue. If any queries, feel free to ask. If you had made a PR, Add up your PR links here.
> 
> hey @prathimacode-hub , i have created a PR #629 . kindly review it.

hello @prathimacode-hub , can you please review the pull request.

**Comment by priyankapiba** [comment]
@adityabisoi can I work on this?

**Comment by adityabisoi** [comment]
Audio or video?

**Comment by priyankapiba** [comment]
Actually on the documentation part, because there isn't any documentation in https://github.com/girlscript/winter-of-contributing/tree/Python/Python/GitHub_Terminologies

**Comment by priyankapiba** [comment]
Which mentor is assigned?

**Comment by adityabisoi** [comment]
You can ping any mentor.
